<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WallWitness</class>
 <widget class="QMainWindow" name="WallWitness">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wall Witness</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="sessionBtn">
       <property name="text">
        <string>Add Session</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="workoutBtn">
       <property name="text">
        <string>Add Workout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="weightBtn">
       <property name="text">
        <string>Add Weight</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSession">
   <property name="text">
    <string>Add Session</string>
   </property>
  </action>
  <action name="actionWorkout">
   <property name="text">
    <string>Add Workout</string>
   </property>
  </action>
  <action name="actionWeight">
   <property name="text">
    <string>Add Weight</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
